lib/ld2410/src/ld2410.cpp

```diff
@@ -261,115 +261,134 @@ bool ld2410::read_frame_()
 	{
 		return false;
 	}
-	if(frame_started_ == false)
-	{
-		uint8_t byte_read_ = radar_uart_ -> read();
-		if(byte_read_ == FRAME_PREFIX_REPORTING)
-		{
-			radar_data_frame_[radar_data_frame_position_++] = byte_read_;
-			frame_started_ = true;
-			ack_frame_ = false;
-		}
-		else if(byte_read_ == FRAME_PREFIX_PROTOCOL)
-		{
-			radar_data_frame_[radar_data_frame_position_++] = byte_read_;
-			frame_started_ = true;
-			ack_frame_ = true;
-		}
-		#if (defined(LD2410_DEBUG_DATA) || defined(LD2410_DEBUG_COMMANDS)) && defined(LD2410_DEBUG)
-		if(debug_uart_ != nullptr)
-		{
-			debug_uart_->print(F("\nRcvd : 00 "));
-		}
-		#endif
-	}
-	else
-	{
-		if(radar_data_frame_position_ < configuration_buffer_size_)
+	const uint32_t _started = millis();
+	int _available = radar_uart_->available();
+	while (_available && (millis() - _started < SERIAL_RECEIVE_MAX_MS)) { // Read for max. N msec
+		yield();
+		if(frame_started_ == false)
 		{
+			uint8_t byte_read_ = radar_uart_ -> read();
```

Comment:
Why spaces here surrounding the `->` ?

Reply:
That's all over that source, didn't do a source reformat (yet), as it will change _every_ line of these files...